ansible/del.vm.centos7.yml

```diff
@@ -0,0 +1,10 @@
+---
+- hosts: localhost
+  connection: local
+  become: no
+  vars_files:
+    - vars.yml
```

Comment:
Do we have _vars.yml_?

Reply:
https://github.com/gluster/samba-integration/blob/samba-build/ansible/vars.yml

Reply:
> https://github.com/gluster/samba-integration/blob/samba-build/ansible/vars.yml

Ah..my bad.